docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md

```diff
@@ -22,7 +22,10 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # How to: Verify that Strings Are in Valid Email Format
-The following example uses a regular expression to verify that a string is in valid email format.  
+The following example is more complex than previous examples and uses a regular expression to verify that a string is in valid email format.  
```

Comment:
Previous here has no referent. The implicit referent is the TOC, but very few people use it for navigation. The original sentence is succinct and to the point; it's best to leave it as is, or at most to change it to "The following example uses a complex regular expression..."